corehq/apps/data_interfaces/migrations/0018_check_for_rule_migration.py

```diff
@@ -3,24 +3,7 @@
 from __future__ import absolute_import
 from django.db import migrations
 
-
-
-def noop(*args, **kwargs):
-    pass
-
-
-def assert_rule_migration_complete(apps, schema_editor):
-    AutomaticUpdateRule = apps.get_model('data_interfaces', 'AutomaticUpdateRule')
-    if AutomaticUpdateRule.objects.filter(migrated=False).count() > 0:
-        raise RuntimeError("""
-            If you are receiving this error in a Dimagi production environment,
-            it is unexpected.
-
-            Otherwise, you have unmigrated AutomaticUpdateRules. You can migrate them
-            by checking out tag `rule-migration` and running:
-
-                python manage.py migrate_rules
-        """)
```

Comment:
This one was on the fence with my self-imposed "one year" rule (it's from September 2018).  Given that it requires you to check out an old branch and run it (something which gets progressively harder over time), I figured I'd delete it.

Aside - this looks like all `AutomaticUpdateRule`s have been deleted, could probably delete that model and associated code now, but I'm not gonna go down that route.